lib/gems/pending/appliance_console/scap.rb

```diff
@@ -2,6 +2,10 @@
 
 module ApplianceConsole
   class Scap
+    def initialize(options)
+      @rules_dir = options[:rules_dir]
+    end
```

Comment:
If the rules_dir is not specified, then doesn't the below code fail?  Should this be a required parameter, and thus not options?